### PR TITLE
Docs corrections

### PR DESCRIPTION
**Pull request by jeffswartz**

### CODE REVIEW

**Comment by aoberoi** [comment]
please see my related comment from the PHP SDK: https://github.com/opentok/OpenTok-PHP-SDK/pull/144#discussion_r61534901

**Comment by aoberoi** [comment]
@jeffswartz i've got similar concerns to those in https://github.com/opentok/Opentok-Java-SDK/pull/98#issuecomment-217040087

File: conf.py [hunk 1]
```diff
@@ -0,0 +1 @@
+sys.path.insert(0, os.path.abspath('.'))
```
Comment (aiham) [inline conf.py]: What's this file for?

File: sample/HelloWorld/static/js/helloworld.js [hunk 2]
```diff
@@ -6,22 +6,10 @@ var publisher = OT.initPublisher('publisher');
...
-    // clients)
-    session.publish(publisher, function(error) {
-      if (error) {
-        console.error('Failed to publish', error);
-      }
-    });
-  },
-
-  // This function runs when another client publishes a stream (eg. session.publish())
+  // This function runs when another client publishes a stream (by calling Session.publish())
```
Comment (aiham) [inline sample/HelloWorld/static/js/helloworld.js]: Would you be able to remove the changes to the sample JS from this PR? We currently have a plan to unify the JS samples in all the different SDKs. I'd prefer to keep this PR focused on the documentation changes. Thanks

**Pull request merged**